specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -9675,28 +10562,23 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/PremierAddOn"
+              "$ref": "#/definitions/FunctionSecrets"
```

Comment:
It is a completely different endpoint. The sitecloneability API is deprecated. The diff just looks odd here. 

---
In reply to: [134407904](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r134407904) [](ancestors = 134407904)